### PR TITLE
feat: allow selection of a ado pipeline yaml

### PR DESCRIPTION
**Pull request by MatthiasKainer**
*tl;dr*

After this change, one can specify the yaml path to the Azure DevOps Pipeline yaml

## Rationale

We have two different usecases that cannot be satisfied with the current setup:

- our pipeline is in a `.azuredevops` folder
- we have projects with multiple pipelines (ie a simple application that has an BFF api and the website)

With the hardcoded specification for the `/azure-pipelines.yaml`, this failed for our templates.

## Solution

By adding an additional field, `yamlPath` to the inputs for the component, which defaults to the previous value, we can satisfy our scenarios, without breaking existing cases. 

## Benefits

1. Allows to place the file in a different folder
2. Enables the usage of multiple pipelines, ie

```yaml
    - id: createApiPipeline
      name: API - Create Azure Pipeline
      action: azure:pipeline:create
      input:
        ...
        yamlPath: "/.azure-pipelines/api.yml"

    - id: runAzurePipeline
      name: API - Run Azure Pipeline
      action: azure:pipeline:run
      input:
        ...
        pipelineId: ${{ steps.createApiPipeline.output.pipelineId }}

    - id: createWebPipeline
      name: Web - Create Azure Pipeline
      action: azure:pipeline:create
      input:
        ...
        yamlPath: "/azure-pipelines/web.yml"

    - id: runWebAzurePipeline
      name: Web - Run Azure Pipeline
      action: azure:pipeline:run
      input:
        ...
        pipelineId: ${{ steps.createWebPipeline.output.pipelineId }}
```

### CODE REVIEW

**Comment by MatthiasKainer** [comment]
hejhej,

any updates? Is there something missing?

Tagging people to bring this up @clemensheithecker 

Best,
m

**Comment by clemensheithecker** [comment]
Looks good to me! Thank you! 👍🎉
